Lib/test/test_xml_etree.py

```diff
@@ -1739,6 +1740,39 @@ def __eq__(self, other):
         self.assertIsInstance(e[0].tag, str)
         self.assertEqual(e[0].tag, 'changed')
 
+    def check_expat224_utf8_bug(self, text):
+        xml = b'<a b="%s"/>' % text
+        root = ET.XML(xml)
+        self.assertEqual(root.get('b'), text.decode('utf-8'))
+
+    def test_expat224_utf8_bug(self):
+        # bpo-31170: Expat 2.2.3 had a bug in its UTF-8 decoder.
+        # Check that Expat 2.2.4 fixed the bug.
+        #
+        # Test buffer bounds at odd and even positions.
+
+        # 2 KiB buffer (even)
```

Comment:
Vice verse. The first test covers the case of buffer bound at odd position, and the second test covers the case of buffer bound at even position.

"2 KiB" is not related. The tests cover all buffer sizes (up to 2 KiB).

Reply:
I removed the comments. "# Test buffer bounds at odd and even positions." is enough.

Reply:
Sorry for my comments were confusing.